src/tests/petsc_smoketest.cpp

```diff
@@ -28,6 +28,8 @@
 
 static char help[] = "Solves 2D Poisson equation using multigrid.\n\n";
 
+#include <algorithm>
```

Comment:
Minor nitpick but wrapping this or moving it to a preexisting SERAC_EDIT_START/ END would help track what we've changed to the example.